packages/inspection-capture-web/src/PhotoCapture/hooks/usePhotoCaptureSightState.ts

```diff
@@ -198,8 +221,14 @@ export function usePhotoCaptureSightState({
     const updatedSightsTaken = [...sightsTaken, selectedSight];
     setSightsTaken(updatedSightsTaken);
     const nextSight = captureSights.filter((s) => !updatedSightsTaken.includes(s))[0];
+    analytics.trackEvent(`sight captured`, {
+      inspectionId,
+      sight: selectedSight.label,
+      nextSight: nextSight?.label ?? 'last-sight',
```

Comment:
same as before : use sight ID instead of label and pass `null` if its the last sight